runtime/src/tests.rs

```diff
@@ -58,16 +58,3 @@ fn must_be_root_to_schedule() {
 		assert_noop!(Scheduler::schedule(RawOrigin::None.into(), 4, None, 0, boxed), BadOrigin);
 	});
 }
-
-#[test]
-fn authority_migration_parity_checks() {
-	use pallet_creditcoin::migrations::v7::{Authorities as AC, SCHEDULER_PREFIX};
-	use pallet_offchain_task_scheduler::Authorities as AT;
-
-	//Pallet prefix
-	let scheduler_prefix = TaskScheduler::name();
-	assert_eq!(SCHEDULER_PREFIX, scheduler_prefix);
-
-	//Storage Prefix
-	assert_eq!(AT::<Runtime>::storage_prefix(), AC::<Runtime>::storage_prefix());
-}
```

Comment:
Not sure why this is removed. Evaluate after we have more answers around the removed migrations.

Reply:
@beqaabu does this still need to be removed ?

Reply:
forgot to reintroduce